src/declaratives/digit-providers/DigitProviders.declarative.jsx

```diff
@@ -79,9 +79,9 @@ const DigitProviders = ({
                                                 {children}
                                             </BrowserRouter>
                                         )}
```

Comment:
I had a problem where I couldn't use `Link` in DigitDialog. I assume this problem originates from `<DigitDialogContextSingletonProvider>` being placed outside the router. Is there some reason why `<DigitDialogContextSingletonProvider>`  this structure is in place or could the provider be moved inside the router?

Reply:
Nope. I will move up the router.